layouts/partials/head.html

```diff
@@ -36,6 +36,32 @@
   })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
   </script>
 {{ end }}
+{{/*  kapa.ai widget
+      TODO: move this to .IsProduction clause ?
+*/}}
+<script>
+document.addEventListener("DOMContentLoaded", function () {
+  var script = document.createElement('script');
+  script.src = 'https://widget.kapa.ai/kapa-widget.bundle.js';
+  script.setAttribute('data-website-id', 'ba5c2cbc-6535-4334-a72e-050f7a681c8a');
```

Comment:
This token should probably live in one of our config-files; and we may want to prevent it being enabled on any domain we don't want it to be running on (including previews? other domains?)

Reply:
move the id to the site config